shotover-proxy/src/transforms/cassandra/sink_single.rs

```diff
@@ -111,8 +117,20 @@ impl CassandraSinkSingle {
                         match timeout(Duration::from_secs(5), results.next()).await {
                             Ok(Some(prelim)) => {
                                 match prelim? {
-                                    (_, Ok(mut resp)) => responses.append(&mut resp),
+                                    (_, Ok(mut resp)) => {
+                                        for message in &resp {
+                                            if let RawFrame::Cassandra(Frame {
+                                                opcode: cassandra_protocol::frame::Opcode::Error,
+                                                ..
+                                            }) = &message.original
+                                            {
+                                                counter!("cassandra_single_failed_request", 1, "chain" => self.chain_name.clone());
+                                            }
+                                        }
+                                        responses.append(&mut resp);
+                                    }
                                     (m, Err(err)) => {
+                                        counter!("cassandra_single_failed_request", 1, "chain" => self.chain_name.clone());
```

Comment:
where is this error coming from?

Reply:
Ah you're right, this is another internal shotover error. I'll remove this counter.

Reply:
did you reply to the wrong comment, this line is still here.

Reply:
Fixed, thought I'd removed it.